components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,17 @@
+class Dropdown {
+  constructor(element) {
```

Comment:
Nice work implementing this dropdown. I hope Dan's solution and lecture were helpful! If you have any questions on this portion, don't be afraid to ask questions to any of the PM's!